gio/Gir.toml

```diff
@@ -1206,6 +1206,9 @@ status = "generate"
     [[object.function]]
     name = "set_child_setup"
     cfg_condition = "unix"
+    [[object.function]]
```

Comment:
we don't do the same for the sys crate?

Reply:
as far as I can see, we are generating all of the `SubprocessLauncher` raw bindings in the `sys` crate, independently if they are unix-specific or not:
https://github.com/gtk-rs/gtk-rs-core/blob/master/gio/sys/Gir.toml

Reply:
The UNIX-specific one should be marked as such in the sys and non-sys crate.

Reply:
yep, will add those to the sys one, thanks